src/Files.App/Interacts/BaseLayoutCommandImplementationModel.cs

```diff
@@ -33,725 +33,722 @@
 
 namespace Files.App.Interacts
 {
-	/// <summary>
-	/// This class provides default implementation for BaseLayout commands.
-	/// This class can be also inherited from and functions overridden to provide custom functionality
-	/// </summary>
-	public class BaseLayoutCommandImplementationModel : IBaseLayoutCommandImplementationModel
-	{
-		#region Singleton
+    /// <summary>
```

Comment:
A big part of formatting of this file has been broken from tabs to spaces. It needs to get back to tabs.

Reply:
It's done.